### PR TITLE
pythonPackages.pysnow: init at 0.7.14

### PR DESCRIPTION
**Pull request by a1mac**
###### Motivation for this change
Adding pysnow into nixpkgs and adding myself as the maintainer for the package.

###### Note
Separate commit with the maintainer info only was denied: https://github.com/NixOS/nixpkgs/pull/79160
I have a few more packages that I would like to add given that this one is accepted

###### Things done
I tested builds with different versions of python on centos with nixpkgs.
I did integration testing with py37 on nixos (nix-shell).

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pysnow/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  pname = "pysnow";
+  version = "0.7.14";
+  
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0a6ce8b5f247fbfe5a53829c2f22391161e88646742283f861bce32bfe1626f1";
+  };
+
+  # https://pypi.org/project/pysnow/0.7.14/
+  disabled = !(isPy27 || pythonAtLeast "3.5");
```
Comment (FRidh) [inline pkgs/development/python-modules/pysnow/default.nix]: We do not support Python < 3.5, so this can be removed

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -357,6 +357,12 @@
     githubId = 5892756;
     name = "Alec Snyder";
   };
+  almac = {
+    email = "alma.cemerlic@gmail.com";
+    github = "a1mac";
+    githubId = 60479013;
+    name = "Alma Cemerlic";
+  };
```
Comment (jonringer) [inline maintainers/maintainer-list.nix]: should be a separate commit:
```
maintainers: add almac
```

File: pkgs/development/python-modules/pysnow/default.nix (hunk 1 above)
Comment (jonringer) [inline pkgs/development/python-modules/pysnow/default.nix]:
```suggestion
```

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -4908,6 +4908,9 @@ in {
 
   pyshp = callPackage ../development/python-modules/pyshp { };
 
+  pysnow = callPackage ../development/python-modules/pysnow { };
+
+
```
Comment (jonringer) [inline pkgs/top-level/python-packages.nix]: extra line
```suggestion
```

**Comment by a1mac** [comment]
Working on fixing it, please don't close. Thanks!

File: maintainers/maintainer-list.nix [hunk 4]
```diff
@@ -366,6 +366,11 @@
       longkeyid = "rsa4096/0x9FAA63E097506D9D";
       fingerprint = "7FDB 17B3 C29B 5BA6 E5A9  8BB2 9FAA 63E0 9750 6D9D";
     }];
```
Comment (jonringer) [inline maintainers/maintainer-list.nix]:
```suggestion
    }];
  };
```

```
$ nix-env -f /home/jon/.cache/nixpkgs-review/pr-79687/nixpkgs -qaP --xml --out-path --show-trace --meta
error: while querying the derivation named '0x0-2018-06-24':
syntax error, unexpected $end, expecting ';', at /home/jon/.cache/nixpkgs-review/pr-79687/nixpkgs/maintainers/maintainer-list.nix:8058:2
```

**Review by jonringer — APPROVED** [review]
diff LGTM
```
[12 built, 210 copied (781.5 MiB), 141.7 MiB DL]
https://github.com/NixOS/nixpkgs/pull/79687
3 package built:
python27Packages.pysnow python37Packages.pysnow python38Packages.pysnow
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build python27Packages.pysnow python37Packages.pysnow python38Packages.pysnow
